src/helpers.ts

```diff
@@ -37,3 +37,21 @@ export function chunkArray<T>(array: T[], size: number) {
   }
   return result;
 }
+
+// converts timestamps from second and microsecond format to milliseconds
+export function anyTimestampToMillis(timestamp: string | number): number {
```

Comment:
Can we add tests for this please?

Reply:
🍻

Reply:
https://github.com/yearn/yearn-sdk/pull/255/commits/b714a3ac9d7d0bcd63c46c71eee0c4800a3047de